es/ponyfill.js

```diff
@@ -0,0 +1,17 @@
+export default function symbolObservablePonyfill(root) {
+	let result;
```

Comment:
I think the only goal here is to support Rollup via ES2015 modules. You don't want to assume any other language features are present, thus requiring users to add ES2015 => ES5 transforms to their build chain to support older browsers.